src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.targets.in

```diff
@@ -46,8 +46,16 @@
     <UsingBrowserRuntimeWorkload Condition="'$(UsingBrowserRuntimeWorkload)' == ''">$(WasmNativeWorkloadAvailable)</UsingBrowserRuntimeWorkload>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(NativeLib)' != ''">
```

Comment:
Are all these properties customer facing our just internal? If just internal or if we are not sure we should prefix with _.